README.MD

```diff
@@ -18,7 +18,7 @@ FERN
 ## Branch convensions
 1. One Branch per Task
 2. Name branch VOL-<task_number>/what-is-being-done, e.g. VLT-001/add-user-authentication
-
+3. Pull Requests will be into 'development-main' branch as 'main' is deployed
```

Comment:
Aren't we deleting the development-main branch and just working straight to main? If so, once this line is removed you can merge

Reply:
Good point